package.json

```
@@ -29,8 +29,15 @@
     "docs:serve": "compodoc -p src/tsconfig.app.json -d docs -s",
     "prepush": "npm run lint:ci",
     "release:changelog": "npm run conventional-changelog -- -p angular -i CHANGELOG.md -s",
-    "cy:open": "npx cypress open ",
-    "cy:run": "npx cypress run"
+    "cy:open": "npx cypress open",
+    "cy:run": "npx cypress run",
+    "cy:my-tests": "npx cypress run -s 'cypress/e2e/my-tests/*' --browser chrome",
+    "reports:merge": "npx mochawesome-merge 'cypress/results/*.json' -o mochawesome.json",
+    "report:html": "npx marge mochawesome.json",
+    "report:all": "npm run reports:merge && npm run report:html",
+    "reports:delete": "rmdir /S /Q cypress\\results && mkdir cypress\\results && rmdir /S /Q mochawesome-report && del mochawesome.json",
+    "cy:run:report": "npm run reports:delete && npm run cy:my-tests && npm run report:all",
```

```suggestion
    "cy:run:report": "npm run reports:delete && npm run cy:my-tests",
```
`npm run report:all` не виконається якщо тести впадуть